src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -244,7 +245,13 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
               showForVariation === 'variation_3' &&
               variation === 'variation_3'
             ) {
-              return <div>scrolling recs</div>;
+              return (
```

Comment:
Would be worth creating a unit tests for this path. Happy to help write them.

Reply:
👍 I will see what I can do